app/controllers/quotes_controller.rb

```diff
@@ -24,7 +31,11 @@ def quote_params
       :total, 
       :comments, 
       :polygons, 
-      :static_map_URL
+      :static_map_URL,
+      :service_expedition_cost,
+      :service_expedition_time,
+      :salt_bags_quantity,
+      :salt_bags_due,
```

Comment:
You don't actually need to send the amount due back to the server... same with the expedition cost. You can (and should) do those calculations on the server based on the `salt_bags_quantity` and `service_expedition_cost`. A malicious client _could_ tamper with the values sent in the form and change those to arbitrary values. Especially when/if you get to the point of actually charging the user via Stripe, you want to make sure to charge them for what _you_ know is the correct amount, not whatever they might have fudged the form to be.

Reply:
done